File: src/main/java/com/anod/appwatcher/utils/Assert.java
package com.anod.appwatcher.utils;

import com.anod.appwatcher.BuildConfig;

/**
 * @author algavris
 * @date 02/05/2017.
 */

public class Assert {

    public static void expr(boolean expr) {
        if (BuildConfig.DEBUG) {
            if (!expr) throw new AssertionError();
        }
    }
}

File: src/main/java/com/anod/appwatcher/utils/Assert.kt
package com.anod.appwatcher.utils

import com.anod.appwatcher.BuildConfig

/**
 * @author algavris
 * *
 * @date 02/05/2017.
 */

object Assert {

    fun expr(expr: Boolean) {
        if (BuildConfig.DEBUG) {
            if (!expr) throw AssertionError()
        }
    }
}
